app/controllers/admin/lookup/advisers_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::Lookup::AdvisersController < Admin::ApplicationController
+  def index
+    @search = ::Lookup::Adviser.ransack(params[:q])
```

Comment:
Would be interested to get to the bottom of this, but for some reason ``Lookup::Firm`` resolves fine within ``Admin::Lookup::FirmsController``, but ``Lookup::Adviser`` doesn't within ``Admin::Lookup::AdvisersController`` and I have to be explicit about the namespace.